### DISCUSSION TITLE
apollo搭建

### DISCUSSION THREAD

**Discussion by suifengershi1027** [apolloconfig/apollo#3785] [Q&A]
搭建apollo时如何屏蔽默认登录页（输入localhost:8070自动重定向到localhost:8070/login然后显示登录页，输入apollo用户密码后会跳转自定义的sso登录页），启动命令改-Dapollo_profile为自定义的登录方式了

**Comment by nobodyiam** [comment]
[实现方式四： 接入公司的统一登录认证系统](https://www.apolloconfig.com/#/zh/development/portal-how-to-implement-user-login-function)

看下是否 `@ConditionalOnMissingProfile` 是否配置正确了
